vars/sectionDeployToEnvironment.groovy

```diff
@@ -37,6 +38,7 @@ def call(params) {
                 sectionInfraBuild(
                   subscription: subscription,
                   environment: environment,
+                  aksSubscription: aksSubscription,
```

Comment:
not used?

Reply:
This one is used - it's passed through to sectionInfraBuild (only from here)

Reply:
yes but it's not used for anything there

Reply:
It's passed again in `params` to https://github.com/hmcts/cnp-jenkins-library/pull/1177/files#diff-2201eb99947401c902306ed743bf8e8e94b57086fa05fb7d193c61ee3af73cfcR8 which is used to check AKS clusters

Reply:
The if check stops that running if somehow those conditions are met but this isn't passed as it fails to get `.name`

Reply:
https://github.com/hmcts/cnp-jenkins-library/pull/1177/files#diff-debf4f95cedccc768d522a59c5a9b2a7e40f2ab05cda7a5d430647976f82ea26R22

Reply:
Ah I see, withAksClient should probably be providing the subscription which would make it clearer but that's ok.